### PR TITLE
Fix for wildcard import resolver, only display interesting files in t…

### PR DESCRIPTION
**Pull request by jsight**
…he report, and some performance tweaks to a query

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/336/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='org.jboss.windup.rules:windup-rulesets' /><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/336/org.jboss.windup.rules$windup-rulesets/testReport'>org.jboss.windup.rules:windup-rulesets</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/336/org.jboss.windup.rules$windup-rulesets/testReport/org.jboss.windup.rules.tests/WindupRulesTest/testWindupRules/'><strong>org.jboss.windup.rules.tests.WindupRulesTest.testWindupRules</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/965/<h2>Failed Tests: <span class='status-failure'>2</span></h2><h3><a name='org.jboss.windup.tests:windup-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/965/org.jboss.windup.tests$windup-tests/testReport'>org.jboss.windup.tests:windup-tests</a>: <span class='status-failure'>2</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/965/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureHibernateTest/testRunWindupHibernate/'><strong>org.jboss.windup.tests.application.WindupArchitectureHibernateTest.testRunWindupHibernate</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/965/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSpringSmallTest/testRunWindupSmallSpringApp/'><strong>org.jboss.windup.tests.application.WindupArchitectureSpringSmallTest.testRunWindupSmallSpringApp</strong></a></li></ul>
Test FAILed.

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/IndexJavaSourceFilesRuleProvider.java [hunk 1]
```diff
@@ -151,7 +151,7 @@ public void perform(GraphRewrite event, EvaluationContext context, JavaSourceFil
             }
             finally
             {
-                WindupWildcardImportResolver.setGraphContext(null);
+                WindupWildcardImportResolver.setContext(null);
```
Comment (lincolnthree) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/IndexJavaSourceFilesRuleProvider.java]: This seems awful, but I'm not sure of a better solution right now.

Comment (jsight) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/IndexJavaSourceFilesRuleProvider.java]: I agree.

**Pull request merged**